src/main/webapp/techassistant.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Seniors Connect</title>
+  <link rel="stylesheet" href="styles/techstyle.css">
+  <script src="script.js"></script>
```

Comment:
Do we need to include script.js?

Reply:
Was not sure if this page would eventually need a script file so I just kept it, but I can remove it.

Reply:
For now it probably isnt necessary to include.  We could add it back later.

Reply:
removed.